deploy/kubernetes/storageclass/ibmc-vpc-file-dp2-StorageClass.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ibmc-vpc-file-dp2
+provisioner: vpc.file.csi.ibm.io
+mountOptions:
+    - hard
+    - nfsvers=4.1
+    - sec=sys
+parameters:
+  profile: "dp2"            # The VPC Storage profile used. https://cloud.ibm.com/docs/vpc?topic=vpc-file-storage-profiles&interface=ui#dp2-profile
+  iops: "100"                       # Default IOPS. User can override from secrets
+  billingType: "hourly"             # The default billing policy used. User can override this default
+  encrypted: "false"                # By default, all PVC using this class will only be provider managed encrypted. The user can override this default
+  encryptionKey: ""                 # If encrypted is true, then a user must specify the encryption key used associated KP instance
+  resourceGroup: ""                 # Use resource group if specified here. else use the one mentioned in storage-secrete-store
+  zone: ""                          # By default, the storage vpc driver will select a zone. The user can override this default
+  tags: ""                          # A list of tags "a, b, c" that will be created when the volume is created. This can be overidden by user
+  classVersion: "1"
+  UID: "0"                           # The initial user identifier for the file share.
+  GID: "0"                           # The initial group identifier for the file share.
```

Comment:
This is lower case too

Reply:
You did introduce this in custom storage class :)